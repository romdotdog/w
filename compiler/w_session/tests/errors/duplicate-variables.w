fn main() {
    let foo = 1;
    let foo = 2;
}