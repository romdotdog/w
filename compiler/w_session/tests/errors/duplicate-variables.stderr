tests/errors/duplicate-variables.w:3:9: error: identifier already defined
    let foo = 2;
        ^^^
